src/app/api/quickbooks/syncLog/syncLog.service.ts

```diff
@@ -81,9 +81,17 @@ export class SyncLogService extends BaseService {
     return await query
   }
 
-  async getOne(conditions: WhereClause) {
+  async getOne(
+    conditions: WhereClause,
+    orderByDirection: 'asc' | 'desc' = 'desc',
+  ) {
     const query = this.db.query.QBSyncLog.findFirst({
       where: conditions,
+      orderBy: [
+        orderByDirection === 'asc'
+          ? asc(QBSyncLog.createdAt)
+          : desc(QBSyncLog.createdAt),
+      ],
```

Comment:
```suggestion
const orderMap = { asc, desc }
...
      orderBy: [orderMap[orderByDirection](QBSyncLog.createdAt)],
```

This cleaner solution could work, can you give it a try?

Reply:
Yes this works. Thanks!